foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
```

Comment:
The TFTP proxy does not need this directory, in fact my TFTP server path is `/var/lib/tftproot`. Would it be possible to get the directory from `/etc/foreman-proxy/settings.d/tftp.yml` ? There's an option `tftproot` in that file that specifies which directory the proxy will use.